internal/user/repository/eventsourcing/eventstore.go

```diff
@@ -611,13 +611,15 @@ func (es *UserEventstore) ProfileByID(ctx context.Context, userID string) (*usr_
 }
 
 func (es *UserEventstore) ChangeProfile(ctx context.Context, profile *usr_model.Profile) (*usr_model.Profile, error) {
+	profile.SetNamesAsDisplayname()
```

Comment:
Shouldn't there be a `OverwriteDisplayname`-func? because the display name can never be changed after it's set initially